misc_docs/syntax/decorator_uncurried.mdx

```diff
@@ -0,0 +1,17 @@
+---
+id: "uncurried-decorator"
+keywords: ["uncurried", "decorator"]
+name: "@@uncurried"
+summary: "This is the `@@uncurried` decorator."
+category: "decorators"
+---
+
+If you have uncurried mode turned off in `rescript.json` and still want to try it on a per-file basis, you can turn it on via 
+
+```rescript
+@@uncurried
+```
+
+at the top of a `.res` file.
+
+_Available since ReScript `11.0.0`._
```

Comment:
Any reference link to our blog post / guide as we do in our other cheatsheets would be useful